config/jobs/kubernetes/wg-k8s-infra/trusted/sig-contribex-k8s-triage-robot.yaml

```diff
@@ -111,12 +114,21 @@ periodics:
         label:lifecycle/rotten
       - --updated=720h
       - --token=/etc/github-token/token
+      - --endpoint=http://ghproxy.default.svc.cluster.local
       - |-
-        --comment=Rotten issues close after 30d of inactivity.
-        Reopen the issue with `/reopen`.
-        Mark the issue as fresh with `/remove-lifecycle rotten`.
+        --comment=The kubernetes project does not have enough contributors to adequately respond to all issues and PRs.
```

Comment:
Do we need contribex to vet this?

Reply:
nit: capitalize `Kubernetes`?

Reply:
That would be why I cc'ed all the contribex leads